reporting/report.go

```diff
@@ -20,16 +21,15 @@ import (
 // a directory named after the selected dataset, or `rita-html-report` if
 // mupltiple were selected, within the current working directory,
 // mongodb must be running to call this command, will exit on any writing error
-func PrintHTML(dbsIn []string, res *resources.Resources) error {
+func PrintHTML(dbsIn []database.RITADatabase, res *resources.Resources) error {
 	if len(dbsIn) == 0 {
 		return errors.New("no analyzed databases to report on")
 	}
 
 	var dbs []string
-	for _, db := range dbsIn {
-		info, err := res.MetaDB.GetDBMetaInfo(db)
-		if err == nil && info.Analyzed {
-			dbs = append(dbs, db)
+	for i := range dbsIn {
```

Comment:
We should maintain the old looping technique if possible. Indexing doesn't look as nice

Reply:
I could go either way. The other way is more inefficient since it makes a string copy for each loop iteration, but this isn't hot code so it doesn't matter much.